manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -1,6 +1,9 @@
 meta:
   environment: ~
 
+  api_domain: (( concat "api." properties.domain ))
+  apps_domain: (( concat "apps." properties.domain ))
```

Comment:
Do we need to change it now to get the smoke tests working? It feels like something that deserves it's own story and a bit of consideration. Especially as we have other SSL stories in-flight at the moment. /cc @bleach